drivers/led/issi/is31fl3733.c

```diff
@@ -97,22 +91,15 @@ void is31fl3733_select_page(uint8_t addr, uint8_t page) {
 void is31fl3733_write_pwm_buffer(uint8_t addr, uint8_t *pwm_buffer) {
     // Assumes page 1 is already selected.
     // Transmit PWM registers in 12 transfers of 16 bytes.
-    // i2c_transfer_buffer[] is 20 bytes
 
     // Iterate over the pwm_buffer contents at 16 byte intervals.
     for (int i = 0; i < IS31FL3733_PWM_REGISTER_COUNT; i += 16) {
-        i2c_transfer_buffer[0] = i;
-        // Copy the data from i to i+15.
-        // Device will auto-increment register for data after the first byte
-        // Thus this sets registers 0x00-0x0F, 0x10-0x1F, etc. in one transfer.
-        memcpy(i2c_transfer_buffer + 1, pwm_buffer + i, 16);
-
 #if IS31FL3733_I2C_PERSISTENCE > 0
         for (uint8_t i = 0; i < IS31FL3733_I2C_PERSISTENCE; i++) {
-            if (i2c_transmit(addr << 1, i2c_transfer_buffer, 17, IS31FL3733_I2C_TIMEOUT) == 0) break;
+            if (i2c_writeReg(addr << 1, i, pwm_buffer + i, 16, IS31FL3733_I2C_TIMEOUT) == 0) break;
```

Comment:
Wont the `i` here be the wrong variable, and not the `IS31FL3733_PWM_REGISTER_COUNT` loop?

Reply:
Yep...just saw that.